.changeset/silver-months-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": patch
+---
+
+Fixed nonce gap bug if in-progress tx sent but TXM shutdown before marked as broadcasted
```

Comment:
nit: Fixed a race condition bug around EVM nonce management, which could cause the Node to skip a nonce and get stuck.